ansible/vault-env

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo $ANSIBLE_VAULT_PASSWORD
```

Comment:
I understand this is a custom variable - not relevant for ansible (the name is similar to names used by ansible for its env vars.)

Reply:
indeed, it's the env var we'll use to configure the password